docs/styleguide/css.rst

```diff
@@ -179,24 +162,68 @@ GOOD:
 .. code::
 
     .product-name {
-        font-style: underline:
-        color: red;
+      font-style: underline:
+      color: red;
     }
 
     .highlighted-box {
-        float: center;
-        width: 30px;
-        background-color: blue;
+      float: center;
+      width: 30px;
+      background-color: blue;
     }
 
 BAD:
 
 .. code::
 
     .product-name {
-        font-style: underline:
-        color: red;
-        float: center;
-        width: 30px;
-        background-color: blue;
+      font-style: underline:
+      color: red;
+      float: center;
+      width: 30px;
+      background-color: blue;
+    }
+
+
+Tailwind and PostCSS
+====================
+
+Tailwind CSS is a utility-first, highly customizable, low-level CSS framework
+that contains the building blocks for building custom designs.
+
+Follow the Tailwind documentation as the primary rule for writing CSS.
+When defining our own classed follow BEM notation witout using abreviations.
+It's ok to use the Tailwind abberviated class, just make sure our custom classes are not abbreviated.
+
+We also use PostCSS to improve the way we manage the CSS:
+
+* Use imports to break CSS into multiple files for development.
+* Used as general minifier and having a single production CSS file. 
+
+GOOD:
+
+.. code::
```

Comment:
This is a big example but there is no description of what is good at it :)

I think that it would help to have a bit of explanation and a review describing how our rules are applied to generate this code.

Reply:
maybe it's good to add this

```
Use one CSS class per line to make it easy manage the diff.
Don't have empty lines between CSS classes as it will fail on Windows.
```

Reply:
Added the info to the rules. I think too many examples are overwhelming  the reader. 
What do you think it looks like after the changes?